src/api/flaskr/service/shifu/admin.py

```
@@ -612,6 +615,21 @@ def _load_billing_order_map(source_bids: Sequence[str]) -> Dict[str, BillingOrde
     return order_map
 
 
+def _collect_operator_user_credit_order_source_bids(
+    ledger_rows: Sequence[CreditLedgerEntry],
+) -> List[str]:
```

Ruff (0.15.13)

[error] Undefined name `List`

(F821)